Gaijinizer patch/Map037.txt

```diff
@@ -409,17 +415,18 @@ me! If you somehow do not go through here ...!#MTLed
 あんな見た目でも、アイツは¨魔王¨なんだ！！
 > CONTEXT: Map037/events/4/pages/0/255/Dialogue/other2
 > CONTEXT: Map037/events/4/pages/1/296/Dialogue/other2
-moron!!It is decided not to force! !
-Even if it looks like that, the guy is 魔 魔 ̈! !#MTLed
+You can't win, you dumb bastard!!
+
+She doesn't look it but she's the demon king!!#Ron
 > END STRING
```

Comment:
Flow:
>_**She doesn't look it, but she's the demon king!!**_

Additionally, I don't know whether to change the formatting of Demon King or try to localize it a bit. Your thoughts? If we don't change it, I think I'd like all instances of it to be capitalized, like "Demon King".

Reply:
Yeah at the time I was also wondering about what exactly to translate it to, opting to default it to the literal translation for the time being. At the very least having a title that matches the gender she appears as would be a good start (e.g. Demon Queen), unless I'm missing some important plot detail that comes up later in the game.

We could go down the Biblical road and call her Lucifier or something since she seems to be in charge of all hell and replacing "historical" figures with cute girls is pretty Japanese, of note is that apparently 魔王 is also used to refer to Satan.

Otherwise considering her apparent age, "Demon Princess" maybe? Though that might imply things about her character that aren't true. Thinking of something snappy and accurate's a bit tricky.